lib/content_scripts/website/pages/series/series.js

```diff
@@ -263,12 +291,12 @@ class SeriesAnimeListLinks {
   constructor() {
     this.seasonsSelect = document.querySelector('.seasons-select');
     if (this.seasonsSelect) {
-      this.seasonNameContainer = this.seasonsSelect.querySelector('h4');
+      this.seasonNameContainer = this.seasonsSelect.querySelector('h4, span[class^=select-trigger__title]');
```

Comment:
It's a `h4` if the series only has one season and a `span[class^=select-trigger__title]` if it has multiple